src/engine/controls/quantizecontrol.cpp

```diff
@@ -11,23 +11,16 @@
 
 QuantizeControl::QuantizeControl(const QString& group,
         UserSettingsPointer pConfig)
-        : EngineControl(group, pConfig) {
-    // Turn quantize OFF by default. See Bug #898213
-    m_pCOQuantizeEnabled = new ControlPushButton(ConfigKey(group, "quantize"), true);
-    m_pCOQuantizeEnabled->setButtonMode(ControlPushButton::TOGGLE);
-    m_pCONextBeat = new ControlObject(ConfigKey(group, "beat_next"));
-    m_pCONextBeat->set(mixxx::audio::kInvalidFramePos.toEngineSamplePosMaybeInvalid());
-    m_pCOPrevBeat = new ControlObject(ConfigKey(group, "beat_prev"));
-    m_pCOPrevBeat->set(mixxx::audio::kInvalidFramePos.toEngineSamplePosMaybeInvalid());
-    m_pCOClosestBeat = new ControlObject(ConfigKey(group, "beat_closest"));
-    m_pCOClosestBeat->set(mixxx::audio::kInvalidFramePos.toEngineSamplePosMaybeInvalid());
-}
-
-QuantizeControl::~QuantizeControl() {
-    delete m_pCOQuantizeEnabled;
-    delete m_pCONextBeat;
-    delete m_pCOPrevBeat;
-    delete m_pCOClosestBeat;
+        : EngineControl(group, pConfig),
+          // Turn quantize OFF by default. See Bug #898213
+          m_pCOQuantizeEnabled(ConfigKey(group, "quantize"), true),
```

Comment:
Minor nit: The `p` prefixis for pointers. 

```suggestion
          m_coQuantizeEnabled(ConfigKey(group, "quantize"), true),
```

Reply:
Done, thanks.